src/wire/Socket.ts

```diff
@@ -13,7 +13,7 @@ import {
 /**
  * Close codes that are deemed to be recoverable by the reconnection policy
  */
-export const recoverableCloseCodes = [1000, 1011];
+export const recoverableCloseCodes = [1000, 1001, 1006, 1011, 1012];
```

Comment:
Can we please use an enum for those?

Reply:
See https://github.com/mixer/tetrisd/blob/1da7eb403a60513764d7fa052f94287c25691952/models/packets.go#L58

Reply:
These were normal ws closure codes, rather than codes specifically sent by tetrisd. But thinking about it, it probably makes more sense to rid of this altogether and just reconnect on any 1xxx error. Only issue there is that Edge serves 1005 for all closures, so that may have to be excluded for compatibility.

Reply:
It still makes sense to define an enum rather than throwing random numbers around.